modules/virt-adding-kernel-arguments-enable-iommu.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-configuring-pci-passthrough.adoc
+
+[id="virt-binding-devices_{context}"]
+= Adding kernel arguments to enable IOMMU for the CPU
+
+To enable IOMMU extension for the BIOS associated with a PCI device, add kernel arguments to Intel or AMD devices. Create the `MachineConfig` object file to add the kernel arguments.
+
+.Prerequisites
+. Administrative privilege to a working OpenShift Container Platform cluster.
+. The CPU belongs to either Intel or AMD brand.
+
+.Procedure
+
+. Create a `MachineConfig` object file that identifies the kernel argument. The following example shows a kernel argument for an Intel CPU.
+
+[NOTE]
+If you have an AMD CPU, specify the kernel argument as `amd_iommu=on`.
+
++
+.Example
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig <1>
+  metadata:
+    labels:
+      machineconfiguration.openshift.io/role: worker <2>
+    name: 100-worker-iommu <3>
+  spec:
+    config:
+      ignition:
+        version: 3.2.0
+    kernelArguments:
+        - intel_iommu=on <4>
```

Comment:
I have not used this, but probably we suggest the below too,
`You should also append the iommu=pt parameter. This will prevent Linux from touching devices which cannot be passed through.`
@vladikr , What do you think ?

Reply:
If this parameter is to be added, we'll need to add some more explanation and context for this parameter. I mean - preventing RHEL from touching devices etc - I can add this info if we can provide a reasonable explanation and some background.